icu4j/main/classes/core/src/com/ibm/icu/number/Precision.java

```diff
@@ -371,6 +371,16 @@ public Precision withMode(MathContext mathContext) {
     /** Package-private clone method */
     abstract Precision createCopy();
 
+    /**
+     * Call this function to copy the fields from the Precision base class.
+     *
+     * Note: It would be nice if this returned the copy, but most impls return the child class, not Precision.
+     */
+    void createCopyHelper(Precision copy) {
```

Comment:
Should this annotated wtih "@internal"? Currently this is a /* package protected */ Should this be a protected method instead? or private?

Reply:
I think package-private is correct. `protected` is actually part of public API since it can be seen by external subclasses. But `private` doesn't work because I need this function in subclasses. I added a comment.